index.bs

```diff
@@ -3754,6 +3781,43 @@ extension for transaction authorization.
     });
 </pre>
 
+## Aborting Authentication Operations ## {#sample-aborting}
+
+The below example shows how a developer may use the AbortSignal parameter to abort an
+ongoing credential registration operation. A similiar procedure applies to an ongoing
+authentication operation. 
+
+<pre class="example" highlight="js">
+    const authAbortController = new AbortController(); 
+    const authAbortSignal = authAbortController.signal;
+
+    authAbortSignal.onabort = function () {
+        // Migrate away from the page and other necessary clean-ups. 
+    }
+
+    var options = {
+        // A list of options. 
```

Comment:
should signal be in `options` like so:
```
   var options = {
       signal: authAbortSignal`,
        // additional options....
   }
```
..?  i.e., [per credman now](https://w3c.github.io/webappsec-credential-management/#credentialrequestoptions-dictionary)?

Reply:
@equalsJeffH I think authenticatorCancel should be sufficient to address the issue for now. We may need a more polished algorithm overall to describe the life cycle. But that is something we can do at PR time. I can collaborate with you and @jyasskin on that.